src/nvim/option_defs.h

```diff
@@ -427,8 +427,8 @@ EXTERN int p_ea;                // 'equalalways'
 EXTERN char_u *p_ep;          // 'equalprg'
 EXTERN int p_eb;                // 'errorbells'
 EXTERN char_u *p_ef;          // 'errorfile'
-EXTERN char_u *p_efm;         // 'errorformat'
-EXTERN char_u *p_gefm;        // 'grepformat'
+EXTERN char *p_efm;         // 'errorformat'
```

Comment:
hmm. Changing type of string option has no precedent. But it should be safe as `char` and `char_u` are guaranteed to alias, so it can be done incrementally for more char/char_u refactors moving forward (thus setting a precedent here).

Reply:
or actually, there already is `char *p_shadafile`